django-data/image/submissions/templates/submissions/submission_validation_summary.html

```diff
@@ -0,0 +1,46 @@
+{% extends "base.html" %}
+
+{% load submissions_filters %}
+
+{% block title %}
+  Validation Summary for submission test
+{% endblock title %}
+
+{% block content %}
+
+{% if validation_summary %}
+  <h1>Validation summary for {{ validation_summary.type }}s</h1>
+  <span class="badge badge-pill badge-primary">Pass: {{ validation_summary.pass_count }}</span>
```

Comment:
You could also write html in `templatetags` and assign `mark_safe` method in order to do stuff with python, as I did for animal/sample. However, I like this representation, nice work :+1:

Reply:
👍